code-challenges.test.js

```diff
@@ -51,10 +137,52 @@ const studyMinutesWeek2 = {
 
 // b) Create the function that makes the test pass.
 
+// Pseudo 
+
+// Input:object 
+// Output:an array of the values sorted from least to greatest
+// Expected:[15, 15, 20, 30, 30, 60, 90],[10, 15, 20, 45, 60, 65, 100]
+
+// Process:
+// Declare a function called objectToArray
+// Isolate all values from the given objects with the method Objects.values into an array
+// Use the .sort() method to organize them in descending order by setting your logic up like: (a, b) => a - b
```

Comment:
I would briefly mention why the logic needs to be set up like that here in your pseudocode. A little bit of a challenge for you to understand what is going on behind the scenes. 😉